app/partials/navigation.jade

```diff
@@ -12,7 +12,7 @@
     .navbar-collapse.collapse(ng-class="{'in bg-blue' : !navCollapsed}", ng-click="navCollapsed=true")
       ul.nav.navbar-nav.navbar-right(ng-show="status.isLoggedIn")
         li.item(ng-show="settings.btcCurrency")
-          a.flex-important.flex-column(ng-click="openBCmarkets()")
+          a.flex-important.flex-column(ng-click="openBCmarkets()" href="")
```

Comment:
Are you sure it's a good idea to put `href=""` everywhere? Doesn't AngularJS do that? Or is there some way to make AngularJS do it automatically?